control-plane/api/v1alpha1/api_gateway_types.go

```diff
@@ -6,12 +6,17 @@ package v1alpha1
 import (
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	gwv1beta1 "sigs.k8s.io/gateway-api/apis/v1beta1"
 )
 
 const (
 	GatewayClassConfigKind = "GatewayClassConfig"
 )
 
+func init() {
```

Comment:
is there a way to do this without an init function? they can make the code more difficult to reason about (though I get if it's not easily doable)

Reply:
I _think_ this is required by Kubebuilder to generate the CRDs @t-eckert might be able to answer this

Reply:
It's unfortunately one of the ugly things about Kubebuilder 😢